v2/guides/blocks/architecture-with-sdk.mdx

```diff
@@ -0,0 +1,29 @@
+
+import { GuideRecipeLink } from "/src/components/guides/GuideRecipeLink"
+import Tabs from '@theme/Tabs';
```

Comment:
Not related to this file, but if someone selects python / golang backend with MFA, then we should show them an error when they click on start guide which will tell them we dont yet support MFA for this yet, and that they will need to use our node SDK alongside the python / golang application server. Have  start guide button in that box / popup which will basically select the "other framework" backend automatically.